docs/process/release.md

```diff
@@ -107,3 +107,52 @@ Releases should be done in "reverse semver order", meaning they should follow:
 Oldest LTS -> Newest LTS -> Patch -> RC -> Next
 
 This can skip any versions which don't need releases, so most weeks are just "Patch -> Next".
+
+## Releasing a new package
+
+Wombat has some special access requirements which need to be configured to publish a new NPM package.
+
+See [this Wombat doc](http://g3doc/company/teams/cloud-client-libraries/team/automation/docs/npm-publish-service#existing-package)
+and [this postmortem](http://docs/document/d/1emx2mhvF5xMzNUlDrVRYKI_u4iUOnVrg3rV6c5jk2is?resourcekey=0-qpsFbBfwioYT4f6kyUm8ZA&tab=t.0)
+for more info.
+
+Angular is _not_ an organization on NPM, therefore each package is published
+independently and Wombat access needs to be managed individually. This also means
+we can't rely on Wombat already having access to a new package.
+
+In order to configure a brand new NPM package, it first needs to be published
+manually so we can add Wombat access to it. Note that this step can and should be
+done prior to weekly releases. The sooner this can be done, the less likely it
+will block the next weekly release.
+
+1.  Check out the `main` branch, which should always have a `-next` version.
```

Comment:
I agree, but I don't think there's an easy way to do that with our build process, since the version is only configured in one place and stamped to all packages during the build. I could include instructions here to overwrite the version in the pre-built artifact to 0.0.0, but that seems sketchy in its own way.

Reply:
Okiedokie, this works for me